src/toil/batchSystems/kubernetes.py

```diff
@@ -52,8 +54,19 @@
 from toil.lib.humanize import human2bytes
 from toil.resource import Resource
 
+from toil.lib.retry import retry
+
 logger = logging.getLogger(__name__)
+     
+def exception_found(e):
+    if e is urllib3.exceptions.MaxRetryError or e is http.client.GATEWAY_TIMEOUT or e is kubernetes.client.rest.ApiException:
```

Comment:
Shouldn't these be `isinstance` checks like [the S3 code has](https://github.com/DataBiosphere/toil/blob/e6dea9602ca05dcf28b190be97673fd57f69022f/src/toil/jobStores/aws/utils.py#L381-L393)? I think `e` is the exception object and not the class of the exception.